### ISSUE TITLE
Glass bottle inconsistency for XP Management VanillaTweaks v1.3.2-beta

### ISSUE THREAD

**Issue by RisPNG** [teakivy/vanilla-tweaks#3]
when right-clicking with a glass bottle onto an enchanting table. it sometimes don't work and sometimes do, when it works, sometimes it gives more bottles than I had currently (1 glass bottle turns into 2 xp bottle). Which can cause glass bottle duplication when smelt in a furnace.

I know its an experimental pack, but it'd be nice to see its fixed. Thank you for your amazing plugin nonetheless.

**Comment by teakivy** [comment]
I couldn't replicate it not working (make sure you have enough xp?) but the other part of this issue should be fixed!

**Comment by RisPNG** [comment]
I tried experimenting with it more and here's what I can gather.

When using /xp set (name) levels or /xp add (name) levels, it won't work, but this one is not a big deal since we gather xp from mining, breed, etc.

When it's working, bottle seems to multiply by 7. so if I have 17 bottles, it messed up, and I got 21 xp bottles instead in return.

Here's a video of me replicating it: https://www.dropbox.com/s/3rch8wo97t1bvcb/2021-08-01_19-18-24.mkv?dl=0
